docker-compose.yaml

```diff
@@ -15,7 +15,7 @@ services:
     container_name: mongo
     networks:
       - shared-network
-    image: mongo
+    image: mongo:3-jessie
```

Comment:
I understand everything but this change?

Reply:
Best practice to pin images rather than relying on `latest` (same reason as with code dependencies). It's even better to pin the image SHA to guarantee version, but went with this more 'user friendly' intermediate step first.